compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -212,6 +218,7 @@ namespace jank::read
     processor::processor(native_persistent_string_view const &f)
       : file{ f }
     {
+      std::setlocale(LC_ALL, "en_US.utf8");
```

Comment:
I don't think it makes sense for us to hard-code the locale. This will break localization for non-English speakers, for example.